internal/pkg/nodestatus/nodestatus.go

```diff
@@ -128,6 +127,17 @@ func (nsf *StatusClient) statusObj(
 	}
 }
 
+func (nsf *StatusClient) statusObjNode(
+	nodeName string,
+) *secprofnodestatusv1alpha1.SecurityProfileNodeStatus {
+	return &secprofnodestatusv1alpha1.SecurityProfileNodeStatus{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      nsf.pol.GetName() + "-" + nodeName,
```

Comment:
could we reuse this in `statusObj` or just use `statusObj` instead?

Reply:
> nice progress, have you had a chance to test the PR manually yet while you're working on the tests?

I changed the implementation, I tested the patch manually, and it works